src/shared/polyfills/document.evaluate.js

```diff
@@ -0,0 +1,4 @@
+'use strict';
+
+const wgxpath = require('wicked-good-xpath');
+wgxpath.install();
```

Comment:
There is no feature detection in these bundles because they are only loaded if they are needed in the first place.